### DISCUSSION TITLE
Suggestion: hide monthly option if no monthly sponsoring is offered

### DISCUSSION THREAD

**Discussion by raphw** [community/community#7935] [Other Feature Feedback, Questions, & Ideas]
Hi, I do not offer monthly sponsoring for legal reasons (https://github.com/sponsors/raphw). Would you consider to not show the option for monthly payments if it's not enabled?

I think it would be better UX as it can be a bit confusing if someone navigates into the tab finding it empty.

Thanks!

**Comment by jlord** [comment]
Great feedback, opening a ticket on our end for this 👍
